src/InsightContainer/index.js

```diff
@@ -78,7 +78,12 @@ const useStyles = makeStyles(({ breakpoints }) => ({
     [breakpoints.up('md')]: {
       margin: 0
     }
-  }
+  },
+  actionButton: {},
```

Comment:
1. Need to pass props: https://github.com/CodeForAfrica/HURUmap-UI/blob/b387f055f0bfc5d83577d8b11946e87c04e3633c/src/InsightContainer/Actions.js#L64
2. Need to override `classes` for each button: https://github.com/CodeForAfrica/HURUmap-UI/blob/b387f055f0bfc5d83577d8b11946e87c04e3633c/src/InsightContainer/Actions.js#L169
3. Since these classes are for `actionButton`, it's best to name them as such: `actionButtonRoot`, `actionButtonIconGrid`, etc.